pandas/_testing/size_tests.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+import pandas as pd
+
+
+def test_size():
```

Comment:
`_testing` is for utility functions used in the tests. Can you move this to `pandas.tests`.

Reply:
I have moved the testing to the pandas tests/generic folder, thank you for the comment!

Reply:
Looks like this test still needs to be removed from this file. Also, I think it should be placed in `tests/frame/methods/test_size.py` and `tests/series/methods/test_size.py`. These will be a new files (they don't exist yet) and require breaking up the test into one for Series and one for DataFrame.